### ISSUE TITLE
Gnome 45.0 - color not applied

### ISSUE THREAD

**Issue by befabry** [vinceliuice/Colloid-gtk-theme#131]
Hello,

First off, thanks for the theme. I have been using this theme for quite some time, always with the same command. 

`./install.sh -t green -c dark --tweaks rimless --tweaks normal --tweaks dracula`

Since today, the color is not applied on the top bar. I cleaned all my themes and did a clean install. Same result.

OS : Arch
Gnome : 45.0

![image](https://github.com/vinceliuice/Colloid-gtk-theme/assets/17452648/7d75a0a4-8296-492f-9936-dab0ecb8d839)

**Comment by vinceliuice** [comment]
Did you installed `User Themes` gnome-shell extension and use gnome-tweaks to set the colloid dark theme ?

**Comment by befabry** [comment]
Yes I did. I have been using this theme for over a year now without problem

**Comment by vinceliuice** [comment]
Did you installed any GDM theme ?

**Comment by befabry** [comment]
Nope, I haven't installed a GDM theme. I only have the Colloid icons,
Borealis cursor themes and this theme.

I pull the themes relatively frequently and reapply them. Never had a
problem until now.

When it first occurred, I tried to use the float option I hadn't noticed to
see what it looks like. It was grey. I proceeded to `rm -rf` all my themes
in `~.themes` then reinstalled it with the first command I gave and it did
not applied the colors. But in the command output and the name of the
themes it mention green.

**Comment by befabry** [comment]
I must add, it is only with the top bar. The color is applied for the
calendar and the menu with Bluetooth, Wi-Fi, whatever
